### PR TITLE
Created BottomNavView. Added Home, Statistics and SettingsFragment. A…

### PR DESCRIPTION
**Pull request by melaniedura**
- Created BottomNavView. 
- Added Home, Statistics and SettingsFragment.
- Added Retrofit to project. 
- Api call to summary endpoint to fetch data for HomeFragment.

### CODE REVIEW

File: data/src/main/java/com/ac03/covid/entites/SummaryData.kt [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+data class SummaryData(
+    @SerializedName("Countries")
+    val countries: List<Country>,
+    @SerializedName("Date")
+    val date: String,
+    @SerializedName("Global")
+    val global: Global
+) {
+    data class Country(
```
Comment (jmanday) [inline data/src/main/java/com/ac03/covid/entites/SummaryData.kt:14]: Why do you place Country class into SummaryData class?

File: data/src/main/java/com/ac03/covid/entites/SummaryData.kt [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        val country: String,
+        @SerializedName("TotalConfirmed")
+        val totalConfirmed: Int,
+        @SerializedName("TotalDeaths")
+        val totalDeaths: Int,
+        @SerializedName("TotalRecovered")
+        val totalRecovered: Int
+    )
+
+    data class Global(
```
Comment (jmanday) [inline data/src/main/java/com/ac03/covid/entites/SummaryData.kt:25]: The same in Global class.

File: app/src/main/res/layout/fragment_home.xml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    tools:context=".ui.HomeFragment">
+
+    <TextView
+        android:id="@+id/textView4"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
```
Comment (jmanday) [inline app/src/main/res/layout/fragment_home.xml:13]: You should use the dimens.xml for that.

File: app/src/main/res/layout/fragment_home.xml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        android:layout_marginStart="16dp"
+        android:textStyle="bold"
+        android:text="GLOBAL DATA" />
```
Comment (jmanday) [inline app/src/main/res/layout/fragment_home.xml:16]: Here strings.xml

File: app/src/main/res/layout/fragment_home.xml [hunk 5]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    tools:context=".ui.HomeFragment">
+
+    <TextView
+        android:id="@+id/textView4"
```
Comment (jmanday) [inline app/src/main/res/layout/fragment_home.xml:10]: The name _textView4_ maybe could be improved.

File: data/src/main/java/com/ac03/covid/api/CovidApiFactory.kt [hunk 6]
```diff
@@ -0,0 +1,25 @@
+package com.ac03.covid.api
+
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object CovidApiFactory {
```
Comment (jmanday) [inline data/src/main/java/com/ac03/covid/api/CovidApiFactory.kt:8]: Why it's called like _Factory_?

**Review by Christianriverabalseras — APPROVED** [review]

**Review by mastalsbota — APPROVED** [review]

**Review by jmanday — APPROVED** [review]

File: data/src/main/java/com/ac03/covid/entites/SummaryData.kt (hunk 1 above)
Comment (melaniedura) [reply data/src/main/java/com/ac03/covid/entites/SummaryData.kt:14]: this was generated with the Json to kotlin class plugin. It doesn't make a difference though, we can have it as a separated class.

File: app/src/main/res/layout/fragment_home.xml (hunk 3 above)
Comment (melaniedura) [reply app/src/main/res/layout/fragment_home.xml:13]: That textview will be removed as the layout needs to be rewritten.

File: data/src/main/java/com/ac03/covid/api/CovidApiFactory.kt (hunk 6 above)
Comment (melaniedura) [reply data/src/main/java/com/ac03/covid/api/CovidApiFactory.kt:8]: Wasn't too sure how to name it. Any suggestions?

**Pull request merged**